scipy/fft/_pocketfft/meson.build

```diff
@@ -7,8 +7,14 @@ pocketfft_threads = []
 fft_deps = []
 if not win_gcc
   if thread_dep.found()
-    pocketfft_threads += ['-DPOCKETFFT_PTHREADS']
     fft_deps += [thread_dep]
+    if not is_windows
```

Comment:
Ah, good point. The code is correct, but the comment is in the wrong place or misphrased. It should say that we _do_ want to rely on pthreads for non-Windows systems, but that for MSVC we _don't_ because likely the threading detection picked up Win32 threading APIs rather than pthreads.

Reply:
Fixed by restructuring the code plus rephrasing the comment.